keyboards/duck/orion/rules.mk

```
@@ -1 +1 @@
-DEFAULT_FOLDER = duck/orion/v3
+DEFAULT_FOLDER = duck/orion/v2_5
```

Honestly, I think it may be better to fail, if the version isn't specified here.
```suggestion
```

Otherwise, may be better to keep using the old value.
```suggestion
DEFAULT_FOLDER = duck/orion/v3
```